src/components/subheader/subheader.js

```diff
@@ -78,6 +78,12 @@ function MdSubheaderDirective($mdSticky, $compile, $mdTheming, $mdUtil) {
         return angular.element(el[0].querySelector('.md-subheader-content'));
       }
 
+      // Set the ARIA attributes on the original element since it keeps it's original place in
+      // the DOM, whereas the clones are in reverse order. Should be done after the outerHTML,
+      // in order to avoid having multiple element be marked as headers.
+      attr.$set('role', 'heading');
+      attr.$set('aria-level', '2');
```

Comment:
This shouldn't be hard-coded to 2; it should be whatever header-level is appropriate for that section of the DOM. We should expose an attribute that lets users specify the level and probably console.log a warning if it's missing.